libs/portal-integration-angular/src/lib/services/theme.service.ts

```diff
@@ -25,15 +25,15 @@ export class ThemeService implements OnDestroy {
 
   public loadAndApplyTheme(themeName: string) {
     return this.http.get<Theme>(`${this.baseUrlV1}/internal/themes/${encodeURI(themeName)}`).pipe(
-      tap((theme) => {
-        this.apply(theme)
+      tap(async (theme) => {
```

Comment:
This is not doing what you think it is doing. From the docs it looks like tap is not waiting for async functions to complete, but this is what you want to do because you want the theme to be applied before you continue. The easiest solution I can come up with without googleing would be using mergeMap. 
Something like this:
```
mergeMap((theme) => {
return from(this.apply(theme)).pipe(map(() => theme)
}
```